kedro/framework/cli/project.py

```diff
@@ -60,6 +60,7 @@
 param_group.param1:value1."""
 INPUT_FILE_HELP = """Name of the requirements file to compile."""
 OUTPUT_FILE_HELP = """Name of the file where compiled requirements should be stored."""
+CONF_SOURCE_HELP = """Path of a directory where project configuration is stored"""
```

Comment:
```suggestion
CONF_SOURCE_HELP = """Path of a directory where project configuration is stored."""
```

Reply:
Thanks for catching this, added in [80ee277](https://github.com/kedro-org/kedro/pull/2117/commits/80ee27733552924408544c57e676e94e9d4fd1f5).